userLogs/models.py

```diff
@@ -1,13 +0,0 @@
-from django.contrib.auth.models import User
-from django.db import models
-
-
-# Create your models here.
-class UserLogs(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
class name should not be plural